.github/workflows/scripts/create-cluster.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+#
+# Creates a cluster on infra.
+#
+
+set -uo pipefail
+
+FLAVOR="$1"
+NAME="$2"
+LIFESPAN="$3"
+WAIT="$4"
+ARGS="$5"
```

Comment:
Must: You call it with: `name, wait, flavor, lifespan, args` in .github/workflows/create-cluster.yml. Here the order is different